packages/user-ui/src/pages/project/page.rs

```diff
@@ -2,14 +2,280 @@ use dioxus::prelude::*;
 use dioxus_logger::tracing;
 use dioxus_translate::Language;
 
-use crate::pages::project::controller;
+use crate::{
+    components::icons::{
+        download::DownloadIcon,
+        pencil::Pencil,
+        person::Person,
+        right_arrow::RightArrow,
+        triangle::{TriangleDown, TriangleUp},
+    },
+    pages::project::controller,
+};
 
 #[component]
 pub fn ProjectPage(lang: Language, project_id: i64) -> Element {
     let ctrl = controller::Controller::init(lang)?;
     let deliberation = ctrl.get_deliberation();
     tracing::debug!("deliberation: {:?}", deliberation);
     rsx! {
-        div { "project page" }
+        div {
+            ProjectProfile { lang }
+            ProjectDetailsMenu {}
+        }
+    }
+}
+
+#[component]
+pub fn ProjectProfile(lang: Language) -> Element {
+    rsx! {
+        div { class: "w-[1300px] h-[300px] mb-[40px] flex flex- row justify-center items-center gap-[40px]",
+            // TODO: connect to data and UI
+            //data section
+            div { class: "w-[720px] h-[260px] flex flex-col justify-center",
```

Comment:
Avoid describe width.
If you need to describe fixed width, you must consider how to scale down into 300px.
Most of mobile viewports will be less than 400px. So 720px component unconditionally break mobile UI.